pom.xml

```diff
@@ -60,6 +61,12 @@
             <version>${playwright.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
That's for changes in testsuite/src/it/java/io/quarkus/ts/startstop/utils/OpenTelemetryCollector.java ?

Reply:
yes